### ISSUE TITLE
Register Alephium chain in WalletConnect Cloud

### ISSUE THREAD

**Issue by nop33** [alephium/alephium-frontend#158]
## TL;DR

For our mobile wallet to appear in the list of wallets in the WalletConnect QR code modal of each dApp we need to register the Alephium chain to WalletConnect.

> If you don't see your chain listed in this [list](https://docs.walletconnect.com/advanced/multichain/chain-list), then you will need to create an issue in GitHub to to get the process started. You can do so by clicking [here](https://github.com/WalletConnect/walletconnect-monorepo/issues/new?assignees=&labels=type%3A+new+chain+request&template=new_chain_to_explorer.md&title=). Once your chain is added to this list, wallets & dapps will be able to indicate support for your chain via WalletConnect's [Cloud](https://cloud.walletconnect.com/).
~~https://docs.walletconnect.com/advanced/multichain/overview~~
https://docs.walletconnect.com/cloud/chains/overview

This is a blocker for this issue: #55 

## Details

To solve point (1) for iOS of this issue (https://github.com/alephium/alephium-frontend/issues/55), I've been working on registering our mobile wallet app to the WalletConnect Explorer, but I am stuck at this step. "Alephium" is not listed in the supported chains:

<details><summary>Click here to see screenshots</summary>

<img width="1728" alt="image" src="https://github.com/alephium/alephium-frontend/assets/1579899/2fd5d289-5a71-485d-bc27-9e728c61ef8d">

![image](https://github.com/alephium/alephium-frontend/assets/1579899/aa39e89b-0d76-43ee-b6ab-944c693199f4)

</details>

## Examples of other chains

https://github.com/WalletConnect/walletconnect-monorepo/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+new+chain+request%22

## TODO
- [ ] Create issue on WalletConnect monorepo with following info
   - [x] JSON-RPC spec for Wallets (only if [one](https://docs.walletconnect.com/2.0/advanced/rpc-reference/ethereum-rpc) doesn't already exist). For more information, please review the Ethereum [JSON-RPC API docs](https://ethereum.org/en/developers/docs/apis/json-rpc/).
   **GitHub Link:**
     - https://github.com/alephium/alephium/blob/master/api/src/main/resources/openapi.json
     - https://github.com/alephium/explorer-backend/blob/master/app/src/main/resources/explorer-backend-openapi.json

   - [x] Write a [CASA namespace spec](https://github.com/ChainAgnostic/namespaces) if not already available for this namespace.
   **GitHub Link:** 
      - https://github.com/ChainAgnostic/namespaces/tree/main/alephium
   - [x] namespaces: known chain namespaces (e.g. eip:155 for Ethereum/EVM-based chains, solana, ...)
      - alephium
   - [x] chains:
   _\*known chains, where the primary key is a compound key composed of namespace + reference (e.g. ETH mainnet is eip155:1)._
   _Please provide labels for each chain (e.g. mainnet, testnet, devnet, ...)_

      Alephium's chain id follows this format `alephium + ":" + group + "_" + network_id` where group can be `1`, `2`, `3` or `4` which represent individual group, or `-1` which represent all groups. The value of network_id could be `mainnet`, `testnet` or `devnet`. Here are a few examples of chain ids:

      ```
      # Alephium mainnet, group 0
      alephium:0_mainnet

      # Alephium testnet, any group
      alephium:-1_testnet
     ```

   - [x] RPC endpoints *list of common/canonical RPC endpoints for the chain(s)
      - https://docs.alephium.org/infrastructure/public-services
   - [x] [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) coin type: * slip44 coin type used in the namespace
      - 1234

**References**
To ensure timely processing of your request, please provide the following references:

- [ ] Source of RPC endpoints (e.g. RPC docs): [link]
- [ ] Source of namespace (if non-EIP155) chain information (particularly chainIds): [link]

**Comment by bumblefudge** [comment]
As for the namespace docs, start [here](https://github.com/ChainAgnostic/namespaces/blob/main/CONTRIBUTING.md) and let me know if i can help! If you know Ergo well enough to help close my pending questions on _their_ PR, that would also be appreciated and perhaps even a stitch in time cuz you'll already know what corner-cases and nits I look for when editing these things😅

**Comment by nop33** [comment]
Now that we have registered Alephium to the CASA Namespace let us try and create an issue on WalletConnect to get listed in their Explorer so that our mobile app is shown in the QR code and iOS UX is improved.

This is the WalletConnect page listing all supported chains: https://docs.walletconnect.com/cloud/chains/chain-list

**Comment by nop33** [comment]
Issue submitted: https://github.com/WalletConnect/walletconnect-monorepo/issues/5283

Thank you @h0ngcha0 🙏 🤜 🤛

**Comment by nop33** [comment]
All Alephium shards have been registered and the Alephium mobile wallet is now listed in the "WalletConnect Explorer": https://walletguide.walletconnect.network/?search=alephium

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/4d192058-8bef-4151-a079-4134d18371e9">

<img width="938" alt="image" src="https://github.com/user-attachments/assets/16420499-815e-451e-8031-1c5b58dfea7d">


Closing issue.
